src/main/java/teammates/ui/webapi/CreateFeedbackQuestionAction.java

```diff
@@ -82,6 +83,8 @@ public JsonResult execute() throws InvalidHttpRequestBodyException {
             return new JsonResult(new FeedbackQuestionData(feedbackQuestion));
         } catch (InvalidParametersException ex) {
             throw new InvalidHttpRequestBodyException(ex);
+        } catch (EntityAlreadyExistsException e) {
+            throw new InvalidOperationException("Trying to create a feedback question that already exists", e);
```

Comment:
Could we reuse the message from the EntityAlreadyExistsException? Since the message is quite similar. Open to discussion on this topic

Reply:
The existing idea is that the business layer (Actions) should throw a business friendly version of the message, but since it is the same, perhaps we could just reuse to avoid defining new literal strings and enforce consistency across Actions

Reply:
I agree with this, @marquestye, do make the change here before we merge this in, thank you!